include/openthread/channel_manager.h

```diff
@@ -87,10 +93,10 @@ uint8_t otChannelManagerGetRequestedChannel(otInstance *aInstance);
 uint16_t otChannelManagerGetDelay(otInstance *aInstance);
 
 /**
- * Sets the delay (in seconds) used for a channel change.
+ * Sets the delay (in seconds) used for a network channel change.
  *
- * The delay should preferably be longer than the maximum data poll interval used by all sleepy-end-devices within the
- * Thread network.
+ * Only available on FTDs. The delay should preferably be longer than the maximum data poll interval used by all
+ * sleepy-end-devices within the Thread network.
```

Comment:
Since you're editing this, can you switch any instance of "sleepy-end-devices" to "Sleepy End Devices" in this file please?